TOOLS/osxbundle.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
```

Comment:
If you're eliminating python2 you might as well remove this line and change `optparse` to `argparse`.

Reply:
The purpose of this pr is not to remove all python2 back compilable codes, but to change the shebang so that python3 can be executed by default. I would like to keep this pr as minimum as possible.

Reply:
Switching to another parsing library is not necessary indeed, but this line should really be removed. It is 100% pointless on python3.

Reply:
`optparse` is a legacy of python2 and has been deprecated since python 3.2. I guess there's no real harm in not switching, but it could be removed at any time so it should be done eventually.

Reply:
@sfan5  fixed, please take a look at it. thanks